### PR TITLE
Add Dlasrt

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

File: native/dlasrt.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+// the elements are sorted in increasing order. If sort == lapack.SortDecreasing,
+// the elements are sorted in decreasing order.
+func (impl Implementation) Dlasrt(s lapack.Sort, n int, d []float64) {
+	checkVector(n, d, 1)
+	d = d[:n]
+	switch s {
+	default:
+		panic("lapack: bad sort")
+	case lapack.SortIncreasing:
+		sort.Sort(sort.Reverse(sort.Float64Slice(d)))
```
Comment (kortschak) [inline native/dlasrt.go]: Can we do this by a normal sort and then a simple slice reverse function written here? That avoids the double interface resolution that what's here costs.

Comment (btracey) [inline native/dlasrt.go]: Sure. I thought this was the idiomatic code based on the example.

Comment (kortschak) [inline native/dlasrt.go]: See what the performance difference is?

Comment (btracey) [inline native/dlasrt.go]: Can we leave it as is now and re-assess when we have the functions that call it in context?  This way we have a better guess on typical sizes.

Comment (kortschak) [inline native/dlasrt.go]: No worries.

**Comment by kortschak** [comment]
LGTM

**Pull request merged**